client/my-sites/site-settings/form-general.jsx

```diff
@@ -346,6 +347,111 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	visibilityOptionsComingSoon() {
+		const { fields, isRequestingSettings, eventTracker, siteIsJetpack, translate } = this.props;
+		const blogPublic = parseInt( fields.blog_public, 10 );
+		const wpcomComingSoon = parseInt( fields.wpcom_coming_soon, 10 );
+
+		return (
+			<FormFieldset>
+				{ ! siteIsJetpack && (
+					<>
+						<FormLabel className="site-settings__visibility-label is-coming-soon">
+							<FormRadio
+								name="blog_public"
+								value="-1"
+								checked={ -1 === blogPublic && 1 === wpcomComingSoon }
+								onChange={ () =>
+									this.handleVisibilityOptionChange( {
+										blog_public: -1,
+										wpcom_coming_soon: 1,
+									} )
+								}
+								disabled={ isRequestingSettings }
+								onClick={ eventTracker( 'Clicked Site Visibility Radio Button' ) }
+							/>
+							<span>{ translate( 'Coming Soon' ) }</span>
+						</FormLabel>
+						<FormSettingExplanation isIndented>
+							{ translate( "Your site is hidden from visitors until it's ready for viewing." ) }
+						</FormSettingExplanation>
+					</>
+				) }
+				{ ! siteIsJetpack && (
+					<>
+						<FormLabel className="site-settings__visibility-label is-public">
+							<FormRadio
+								name="blog_public"
+								value="1"
+								checked={ blogPublic === 0 || blogPublic === 1 }
+								onChange={ () =>
+									this.handleVisibilityOptionChange( {
+										blog_public: 1,
+										wpcom_coming_soon: 0,
+									} )
+								}
+								disabled={ isRequestingSettings }
+								onClick={ eventTracker( 'Clicked Site Visibility Radio Button' ) }
+							/>
+							<span>{ translate( 'Public' ) }</span>
+						</FormLabel>
+						<FormSettingExplanation isIndented>
+							{ translate(
+								'Your site is visible to everyone, and may be indexed by search engines.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **45** times:
`translate( 'Your site is visible to everyone, and it may be indexed by search engines.' )` *ES Score: 9*